### ISSUE TITLE
Require images

### ISSUE THREAD

**Issue by Stalso** [Hashnode/mern-starter#164]
Hello. I am inspecting you v2.0.0 branch. How do you require images on server side in development? As far as I understand, in production mode webpack bundles all this stuff for you. But how you deal with it in development? I have noticed, that you do not use webpack-isomorphic tools or something similar.

**Comment by ayoola-solomon** [comment]
👍 on these... been trying to use `webpack-isomorphic tools` but keeps on getting `[webpack-isomorphic-tools] (waiting for the first Webpack build to finish)`

**Comment by Stalso** [comment]
Maybe it is good idea to implement server webpack build during development,  like it is done in [this](https://github.com/choonkending/react-webpack-node) template.  They make webpack build of server and run nodemon on compiled code

**Comment by somus** [comment]
Fixed in ffe1f5e45312bca76c53dbb191cde25a8b491b24. Check the commit to see usage in both js and css files.

Instead of webpack-isomorphic-tools(I faced same problem as @ayoola-solomon), I've used [babel-plugin-webpack-loaders](https://github.com/istarkov/babel-plugin-webpack-loaders) to load assets. In production, I'm moving assets to `dist` folder and using [fake-url-loader](https://www.npmjs.com/package/fake-url-loader) to get the correct file names.
